index.html

```diff
@@ -0,0 +1,686 @@
+                        <div class="funding-thumbnail-bottom">
+                            <p class="funding-thumbnail-bottom-text"><a class="funding-thumbnail-bottom-text" href="#">SG’s cashback app ShopBack closes $75m Series B led by Temasek </a></p>
+                            <h5 class="funding-thumbnail-bottom-text"><a class="funding-thumbnail-bottom-text" href="#">on DealStreetAsia</a></h5>
+                        </div>
+                    </div>
+                    <div class="funding-thumbnail">
+                        <div class="funding-thumbnail-head">
+                            <img class="funding-thumbnail-head-img" src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F818394773895581697%2FY3dtwEG0.jpg&signature=5612aa580ccbd58837448b0d910a15cc" alt="funding logo">
+                            <div class="funding-thumbnail-head-right">
+                                <h4 class="funding-thumbnail-company"><a class="funding-thumbnail-company" href="#">Shopback</a></h4>
+                                <div class="funding-thumbnail-head-right-bottom">
+                                    <h4 class="funding-thumbnail-price">$30M</h4>
+                                    <h4 class="funding-thumbnail-source">SERIES B</h4>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="funding-thumbnail-middle">
+                            <p class="funding-thumbnail-middle-p"><i class="fas fa-tag"></i> E-Commerce, +3</p>
+                            <p class="funding-thumbnail-middle-p"><i class="fas fa-briefcase"></i> Rakuten, +5</p>
+                        </div>
+                        <div class="funding-thumbnail-bottom">
+                            <p class="funding-thumbnail-bottom-text"><a class="funding-thumbnail-bottom-text" href="#">SG’s cashback app ShopBack closes $75m Series B led by Temasek </a></p>
+                            <h5 class="funding-thumbnail-bottom-text"><a class="funding-thumbnail-bottom-text" href="#">on DealStreetAsia</a></h5>
+                        </div>
+                    </div>
+                </div>
+        </div> 
+
+        <div class="rubric general-container">
+
+                <div class="rubric-thumbnail">
+                    <h2 class= "section-title"><a class="section-title-a" href="#">Apps ></a></h2>
+
+                    <div class="rubric-background-container">
+                        <div class="second-tile-gradient">
+                        </div>
+                        <div class="rubric-thumbnail-head">
+                            <h4 class="most-popular">MOST POPULAR</h4>
+                            <h2><a class="rubric-first-title" href="#">Logitech built a 3-in-1 charger for Apple fans</a></h2>
+                            <h5 class="rubric-second-title"><a class="rubric-second-title" href="#">Napier Lopes</a> - 20 hours ago</h5>
```

Comment:
- [ ] All the articles in this section should rearrange in the tablet version to two in a row. Your page shows three at some point. This is also related to the media query, so check your px breaks.

- [ ] When changing to the mobile version, your articles are displaying in two columns when they should occupy 100% width. Check the breakpoints to match the original page!

- [ ] In the mobile version your titles are not sized appropriately (The main title is too small and the red titles too large).  Check the font size and make sure they are also responsive.